.gitignore

```diff
@@ -0,0 +1,2 @@
+# ignore binaries
+beagle-tester
```

Comment:
Makes sense. Might be useful to have some binaries place in releases. Could do more to improve release process, which mostly depends on the buildroot image builds.

Reply:
What do you suggest we should do for release process?
I was thinking as it's a binary we can just release that with an `install.sh` file?

Reply:
It gets handled by debian packaging for the standard debian distro and by buildroot separately. No need for install.sh. 'make install' is good enough.